ansible-scylla-node/defaults/main.yml

```diff
@@ -274,6 +274,21 @@ scylla_ssl:
   client:
     enabled: true
 
+# Two types of encryption keys are available: System Keys and Table Keys.
+
+# System keys can be used for encrypting system data, like commit logs, batches and hints logs.
+# However, in the context of table keys (for encrypting sstables), if the user decides to use
+# a Replicated Key Provider, a system key will also be necessary for encrypting the encrypted_keys table,
+# where the replicated keys will be kept.
+# If you want to use this role for managing your system keys, regardless of them being used for system_info encryption
+# or for encrypting the encrypted_keys table, you should set {{ handle_system_keys }} to true and place all of your system
+# keys under the {{ localhost_system_key_directory }} folder. The role will copy all the local keys to the {{ system_key_directory }}
```

Comment:
> you have a per-host sub-directories structure there, don't you?

we do because the `inventory_hostname` is part of `localhost_table_key_directory` and `localhost_system_key_directory` by default:
```
localhost_system_key_directory: "{{ inventory_dir }}/encryption_at_rest/{{ inventory_hostname }}/system_encryption_keys"
localhost_table_key_directory: "{{ inventory_dir }}/encryption_at_rest/{{ inventory_hostname }}/table_encryption_keys"
```

Note that based on that the description is not wrong.
I added a comment clarifying that right below the statement that you mentioned.

Reply:
Done